python/examples/minimal_build/README.md

```diff
@@ -24,32 +24,32 @@ an eye toward maximum portability across different Linux distributions. This
 may help for contributors debugging build issues caused by their local
 environments.
 
-## Fedora 31
+## Fedora 35
```

Comment:
Note: we should perhaps test those minimal builds as part of our nightly builds, but that's outside the scope of this PR.